packages/generator/test/commonTemplateHelper.test.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from "chai";
+import { addNamespace } from "../src/commonTemplateHelper";
+
+describe("When adding namespaces to individual types", () => {
+  it("Should prefix the types namespace when provided with any string ", () => {
```

Comment:
```suggestion
  it("prefixes any string with a namespace ", () => {
```
Per our best practices / style guide, test descriptions should be `it("does a thing")`, not `it("should do a thing")`.